cmd/manager/main.go

```diff
@@ -271,6 +272,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&atlasbackupcompliancepolicy.AtlasBackupCompliancePolicyReconciler{
+		Scheme:                      mgr.GetScheme(),
+		Client:                      mgr.GetClient(),
+		EventRecorder:               mgr.GetEventRecorderFor("AtlasBackupCompliancePolicy"),
+		GlobalPredicates:            globalPredicates,
+		Log:                         logger.Named("controllers").Named("AtlasBackupCompliancePolicy").Sugar(),
+		AtlasProvider:               atlasProvider,
```

Comment:
@helderjs for easier testing we could place the interfaces here directly as needed.

@roothorp not a comment for this PR in particular, we probably need a wider discussion before going this way anyway.

Reply:
Resolving to track in CLOUDP-252792